### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by alexsomai**
CC @ikedam @oleg-nenashev @martinda

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Alexandru Somai and Jenkins contributors
```
Comment (alexsomai) [inline LICENSE.md]: I don't know exactly how this should be done.
Should I also mention explicitly other contributors to this project? Ikeda Yasuyuki and Alan Harder?

Comment (oleg-nenashev) [inline LICENSE.md]: Makes sense to do it since a significant amount of code has been taken from Copy Artifact plugin

**Comment by oleg-nenashev** [comment]
LGTM, although ordering of license holders is being commonly handled in a historical order. Not very important IMHO

**Comment by alexsomai** [comment]
Ok, I can change the order

**Comment by alexsomai** [comment]
Should I also change the year? like `2004 - 2016`? (I think that 2004 is the first year that appears in the files)

**Comment by ikedam** [comment]
I don't know much about licenses and I'm not sure how to do for the license file.

**Comment by ikedam** [comment]
+1 for `2004-2016`.

**Comment by martinda** [comment]
:+1:

**Pull request merged**